cl/internal/llvm/llvm.go

```diff
@@ -0,0 +1,16 @@
+package llvm
```

Comment:
don't create this package. just use https://pkg.go.dev/github.com/goplus/llgo/cpp/llvm